sections/introduction.include

```diff
@@ -572,21 +574,21 @@
   the text "Sample page". Similarly, the <{body}> element contains an <{h1}> element, a <{p}>
   element, and a comment.
 
-  <hr />
+  <hr>
```

Comment:
I'm happy with this but I thought we were adding the closing tag? (same as img). I can update the style guide if we choose this syntax